data/json/recipes/recipe_food.json

```diff
@@ -4560,9 +4560,30 @@
     "tools": [ [ [ "surface_heat", 100, "LIST" ] ], [ [ "pot_canning", -1 ] ] ],
     "components": [ [ [ "jar_glass", 1 ] ], [ [ "bone_human", 20 ] ], [ [ "water", 12 ], [ "water_clean", 12 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "meat_salted",
+    "id_suffix": "vacuum_packed",
+    "container": "plastic_bag_vac",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_MEAT",
+    "skill_used": "cooking",
+    "difficulty": 4,
+    "time": 20000,
+    "autolearn": true,
+    "qualities": [ { "id": "CUT", "level": 1 }, { "id": "COOK", "level": 3 } ],
+    "tools": [ [ [ "vac_sealer", 10 ], [ "makeshift_sealer", 20 ] ] ],
+    "components": [
+      [ [ "water", 1 ], [ "water_clean", 1 ] ],
+      [ [ "bag_plastic", 1 ] ],
+      [ [ "meat", 2 ] ],
+      [ [ "salt_water", 1 ], [ "saline", 5 ], [ "salt", 10 ], [ "chem_saltpetre", 5 ] ]
+    ]
+  },
   {
     "type": "recipe",
     "result": "bag_meat_vac",
+    "obsolete": true,
```

Comment:
Mark these recipes as obsolete instead of deleting them so they do not display the warning when a user migrates to this version. These recipes should later be deleted.